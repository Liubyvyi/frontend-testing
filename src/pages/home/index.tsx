import React from "react"
import Head from "next/head"

export const Home = () => {
  return (
    <div className="container">
      <Head>
        <title>Home</title>
      </Head>

      <div>Home</div>

      <style jsx>{`
        .container {
          min-height: 100vh;
          padding: 0 0.5rem;

          display: flex;
          align-items: center;
          flex-direction: column;
          justify-content: center;
        }
      `}</style>
    </div>
  )
}
