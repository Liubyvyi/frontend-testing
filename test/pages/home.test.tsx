import React from "react"
import { render } from "@testing-library/react"

import { Home } from "@pages/home"

describe("Home page", () => {
  it("should match snapshot", () => {
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })
})
