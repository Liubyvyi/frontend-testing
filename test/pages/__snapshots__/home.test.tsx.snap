// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render 1`] = `
<div>
  <div
    class="jsx-2631949316 container"
  >
    <div
      class="jsx-2631949316"
    >
      Home
    </div>
  </div>
</div>
`;
